src/layouts/Layout.astro

```diff
@@ -20,16 +20,22 @@ const { title } = Astro.props;
 	</body>
 </html>
 <style is:global>
-	:root {
-		--accent: 124, 58, 237;
-		--accent-gradient: linear-gradient(45deg, rgb(var(--accent)), #da62c4 30%, white 60%);
-	}
 	html {
 		font-family: system-ui, sans-serif;
-		background-color: #F6F6F6;
+		background-color: #212121;
+		color: #cfb39d;
+		height: 100%;
+	}
+	body {
+		height: 100%;
 	}
```

Comment:
Can be replaced with `<body class="h-full">...</body>`

Reply:
Changes added